doc/developers/performance.rst

```diff
@@ -325,9 +325,11 @@ standalone function in a ``.pyx`` file, add static type declarations and
 then use Cython to generate a C program suitable to be compiled as a
 Python extension module.
 
-The official documentation available at http://docs.cython.org/ contains
-a tutorial and reference guide for developing such a module. In the
-following we will just highlight a couple of tricks that we found
+The official documentation available at http://docs.cython.org/ contains a
+tutorial and reference guide for developing such a module. For more information
+see :ref:`cython`.
+
+In the following we will just highlight a couple of tricks that we found
```

Comment:
This section feels a bit weird. It's named `Performance tips for the Cython developer` but we actually don't give tips. Instead we link to another page that give tips. I think redirection is fine but we should probably change the name of the section

Reply:
Also the sub section ``Using OpenMP`` should probably move into the new cython page